set.mm

```diff
@@ -649101,15 +649118,15 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
        axioms over ~ df-sub in certain situations.  Theorem ~ resubval shows
        its value, ~ resubadd relates it to addition, and ~ rersubcl proves its
        closure.  It is the restriction of ~ df-sub to the reals: ~ subresre .
-       (Contributed by Steven Nguyen, 7-Jan-2022.) $)
+       (Contributed by Steven Nguyen, 7-Jan-2023.) $)
     df-resub $a |- -R = ( x e. RR , y e. RR |->
                        ( iota_ z e. RR ( y + z ) = x ) ) $.
   $}
 
   ${
     $d x y z A $.  $d x y z B $.
     $( Value of real subtraction, which is the (unique) real ` x ` such that
-       ` B + x = A ` .  (Contributed by Steven Nguyen, 7-Jan-2022.) $)
+       ` B + x = A ` .  (Contributed by Steven Nguyen, 7-Jan-2023.) $)
     resubval $p |- ( ( A e. RR /\ B e. RR ) ->
                    ( A -R B ) = ( iota_ x e. RR ( B + x ) = A ) ) $=
```

Comment:
The proof is still very long. Maybe some theorems/lemmas can be extracted.

Reply:
It looks like this comment is non-blocking, so I will merge.  But I agree with @avekens that there may be a possibility to extract meaningful lemma(s) to ease understanding of this huge proof.